python/aether_routines.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from datetime import datetime
+from datetime import timedelta
+import numpy as np
+from netCDF4 import Dataset
+
+#-----------------------------------------------------------------------------
+# Get some ancillary information from the netcdf file.  This simply
+# mirrors the gitm header info.
+# Inputs:
+#  - filelist: a list of netcdf files. Really, only the last file is used.
+#              also, the number of files is recorded.
+# Output:
+#  - header: A dictionary containing information about the netcdf file, such
+#            as nLons, nLons, nAlts, nVars, variable names, time(s)
+#-----------------------------------------------------------------------------
+
+def read_aether_header(filelist):
+
+    header = {}
+    header["nFiles"] = len(filelist)
+    header["version"] = 0.1
+    header["nLons"] = 0
+    header["nLats"] = 0
+    header["nAlts"] = 0
+    header["nVars"] = 0
+    header["vars"] = []
+    header["time"] = []
+    header["filename"] = []
+
+    header["filename"].append(filelist[-1])
+    
+    ncfile = Dataset(filelist[-1],'r')
```

Comment:
```suggestion
    ncfile = Dataset(filelist[-1], 'r')
```

Looks like this needs to be linted.

Reply:
Also, if you use `with`, file closure happens cleanly even if the code errors.

Reply:
I have not used "with" and am not sure how to use it.  I will investigate this.  My guess is that this python code will not be super important as others develop better ones....  This is just sort of a start.